webhooks.go

```diff
@@ -123,7 +123,12 @@ type WebhookPayload struct {
 
 // Use this method to parse the webhook signature given as JSON in the webhook response
 func (mg *MailgunImpl) VerifyWebhookSignature(sig Signature) (verified bool, err error) {
-	h := hmac.New(sha256.New, []byte(mg.APIKey()))
+	key := mg.WebhookSigningKey()
+	// For backwards compatibility, fall back to the api key
+	if key == "" {
+		key = mg.APIKey()
+	}
```

Comment:
Could you move this fall back into `(mg *MailgunImpl) WebhookSigningKey()` for DRY?

Reply:
For sure, thanks for the quick review!